### PR TITLE
Docs tasklist links

### PR DESCRIPTION
**Pull request by christinaausley**
## What is the purpose of the change

Resolves link complaints in https://github.com/camunda/camunda-platform-docs/actions/runs/4659462943/jobs/8246377412 based on [Slack](https://camunda.slack.com/archives/C026U8GBNSW/p1680893718374489?thread_ts=1680867949.253929&cid=C026U8GBNSW) conversation.

## PR Checklist

- [x] My changes apply to an already released version, and I have added them to the relevant `/versioned_docs` directory, or they are not for an already released version.
- [x] My changes apply to future versions, and I have added them to the main `/docs` directory, or they are not for future versions.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Review by akeller — DISMISSED** [review]

**Review by pepopowitz — APPROVED** [review]
Thank you for fixing these all!!!!

**Review by akeller — DISMISSED** [review]

**Comment by akeller** [comment]
Following guidance from @pepopowitz on #1941, @christinaausley please re-enable errors on broken markdown links.

File: docusaurus.config.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ module.exports = {
   url: "https://docs.camunda.io",
   // baseUrl: "/camunda-cloud-documentation/",
   baseUrl: "/",
-  onBrokenLinks: "warn",
+  onBrokenLinks: "throw",
```
Comment (pepopowitz) [inline docusaurus.config.js:10]: 🎉 🙌 💪

**Review by pepopowitz — APPROVED** [review]
Yay!!!

**Pull request merged**